modules/carto/src/api/layer-map.ts

```diff
@@ -110,7 +107,7 @@ export function getLayer(
     },
     geojson: {
       Layer: GeoJsonLayer,
-      defaultProps: {lineWidthScale: 2}
+      defaultProps: {}
```

Comment:
As `defaultProps` is empty you can simply remove it

Reply:
I was not sure if I could remove it. Done